adoc/admin-gpus.adoc

```diff
@@ -0,0 +1,163 @@
+[[gpus]]
+= NVIDIA GPUs
+
+SUSE's CaaS Platform supports scheduling GPU-dependent workloads on NVIDIA GPUs as a technical preview. This can be a powerful way to run compute-intensive workloads such as machine learning pipelines. This section illustrates how to prepare your host machine to expose GPU devices to your containers, and how to configure Kubernetes to schedule GPU-dependent workloads.
+
+== Prepare the host machine
+
+=== Install the GPU drivers
+
+Not every worker node in the cluster need have a GPU device present. On the nodes that do have one or more NVIDIA GPUs, install the drivers from NVIDIA's repository.
+
+[source,bash]
+----
+# zypper addrepo --refresh https://download.nvidia.com/suse/sle15sp2/ nvidia
```

Comment:
The driver source is NVIDIA, this is unavoidable, we cannot repackage the driver and support it ourselves